src/internal/database/migrations/migration-transformer/disable-concurrent-index-transformer.ts

```diff
@@ -0,0 +1,25 @@
+import { Migration } from 'postgres-migrations/dist/types'
```

Comment:
i would simple name the folder `transformers` since we are already in the `migrations` folder